scotty.cabal

```diff
@@ -64,81 +63,87 @@ Extra-source-files:
     examples/static/jquery-json.js
     examples/uploads/.keep
 
-Library
-  Exposed-modules:     Web.Scotty
-                       Web.Scotty.Trans
-                       Web.Scotty.Internal.Types
-  other-modules:       Web.Scotty.Action
-                       Web.Scotty.Route
-                       Web.Scotty.Util
-  default-language:    Haskell2010
-  build-depends:       aeson                 >= 0.6.2.1  && < 2.2,
```

Comment:
I'm kinda skeptical that these bounds really make a lot of sense. How were they calculated?

In the PR, I only bound `base` and leave it up to Cabal. To compensate for lack of bounds in the cabal file, I've included freeze files from some successful builds with a variety of GHCs. If people are adamant about it, I can use the feeze files to compute version bounds.